lib/Routing/Dynamic/DataObjectRouteHandler.php

```diff
@@ -57,8 +57,8 @@ public function __construct(
      */
     public function getRouteByName(string $name)
     {
-        if (preg_match('/^data_object_(\d+)_(.*)$/', $name, $match)) {
-            $slug = DataObject\Data\UrlSlug::resolveSlug($match[2]);
+        if (preg_match('/^data_object_(\d+)_(\s+)_(.*)$/', $name, $match)) {
```

Comment:
```suggestion
        if (preg_match('/^data_object_(\d+)_(\d+)_(.*)$/', $name, $match)) {
```
Since we use the id of the site `\d+` should be sufficient.